README.md

```diff
@@ -43,14 +43,13 @@ Chromium, FireFox, and IE:
 
   * **Limited Memory.** All except the very latest, expensive CE devices have a
     very small amount of memory available for applications. This usually is
-    somewhere in the ballpark of 200MB-500MB, including graphics and media
+    somewhere in the ballpark of 500MB, including graphics and media
     memory, as opposed to multiple gigabytes of CPU memory (and more gigabytes
     of GPU memory) in modern desktop and laptop computers, and mobile devices.
   * **Slow CPUs.** Most CE devices have much slower CPUs than what is available
     on even a budget desktop computer. Minor performance concerns can be greatly
-    exaggerated, which seriously affects priorities.
-  * **Fewer cores.** CE System-on-a-Chip (SoC) processors often do not have as
-    many processor cores as we are used to in modern computers.
+    exaggerated, which seriously affects priorities. Cobalt currently expects a
+    4-core 32-bit ARMv7 CPU as a baseline.
```

Comment:
Is this in the hardware requirements? I don't like putting a hard number in here, because i.e. if we at some point require a minimum of 8 cores, this section probably goes unnoticed and confused people

Reply:
Yes it is in HW requirements, and it changes very, very rarely. We only have ever changed it once ( from single core and allowing MIPS ), and every forecast and silicon roadmap i've looked at will keep it at 4-core ARMv7 for next 5 years at least.
I do like declaring the target segment pretty loudly up front - gives readers immediate context of what this project is aiming at. I'd rather keep this in